plugin/src/test/resources/jenkins/plugins/openstack/compute/JCloudsCloudTest/globalConfigMigrationFromV1/config.xml

```diff
@@ -29,6 +29,7 @@
           <imageId>RDU2/ac98e93d-34a3-437d-a7ba-9ad24c02f5b2</imageId>
           <hardwareId>RDU2/16</hardwareId>
           <labelString>label</labelString>
+          <mode>EXCLUSIVE</mode>
```

Comment:
Do not add it here. The test verified the config is correctly migrated from the old format.

Reply:
Ah, thanks for pointing that out.  I've removed it now.